frontend/components/settings/EmailChange.vue

```diff
@@ -0,0 +1,313 @@
+                            type="password"
+                            class="focus-ring-1 w-2/3 rounded-md border-2 border-natural-300 bg-natural-50 py-0.5 pl-2 pr-1 text-text placeholder:text-text hover:border-calypso-400 focus:border-calypso-400 focus:outline-none dark:border-natural-700 dark:bg-natural-900 dark:text-natural-50 dark:hover:border-calypso-400 dark:focus:border-calypso-400"
+                            @keyup.enter="onSubmit"
+                        />
+                        <span
+                            class="mb-3 flex w-2/3 justify-start text-sm text-mahagony-600 dark:text-mahagony-300"
+                            >{{ errors.password }}</span
+                        >
+                    </div>
+                </div>
+                <div class="flex w-full justify-center pb-4 pt-6">
+                    <button
+                        class="w-44 rounded-md border-2 border-dandelion-300 bg-natural-50 px-2 py-0.5 text-base font-medium hover:bg-dandelion-200 dark:border-dandelion-300 dark:bg-natural-900 dark:text-natural-50 dark:hover:bg-pesto-600"
+                        @click="onSubmit"
+                    >
+                        <T key-name="dashboard.user.settings.change.email" />
+                    </button>
+                </div>
+            </div>
+        </Dialog>
+        <Sidebar
+            v-model:visible="isVisible"
+            modal
+            position="right"
+            :auto-z-index="true"
+            :draggable="false"
+            class="z-50 mt-auto flex h-[95%] w-full flex-col rounded-t-md bg-background font-nunito dark:bg-background-dark sm:hidden sm:w-4/5 md:rounded-xl lg:-z-10"
+            :pt="{
+                root: {
+                    class: 'font-nunito bg-background dark:bg-background-dark z-10 lg:-z-10 lg:hidden ',
+                },
+                header: {
+                    class: 'flex justify-start pb-2 pl-9 font-nunito bg-background dark:bg-background-dark dark:text-natural-50 rounded-3xl',
+                },
+                title: {
+                    class: 'font-nunito text-4xl font-semibold',
+                },
+                content: {
+                    class: 'font-nunito bg-background dark:bg-background-dark px-0 -ml-2 sm:pr-12 h-full',
+                },
+                footer: { class: 'h-0' },
+                closeButton: {
+                    class: 'justify-start w-full h-full items-center collapse',
+                },
+                mask: {
+                    class: 'sm:collapse bg-natural-50',
+                },
+            }"
+            @hide="close"
+        >
+            <template #header>
+                <button class="-ml-6 flex justify-center pr-4" @click="close">
+                    <span class="pi pi-angle-left text-3xl" />
+                </button>
+                <div class="font-nunito text-4xl font-semibold">
+                    <T key-name="dashboard.user.settings.email.change" />
```

Comment:
![image](https://github.com/user-attachments/assets/73291899-923e-489e-a307-c1c9975eede4)
Make it not split. The text should not break